build/lib/builtInExtensions.ts

```diff
@@ -15,8 +15,6 @@ import * as fancyLog from 'fancy-log';
 import * as ansiColors from 'ansi-colors';
```

Comment:
It appears like the transpiled file `build/lib/builtInExtensions.js` needs to be committed.

CI Failure: https://dev.azure.com/vscode/VSCode/_build/results?buildId=131114&view=logs&j=c7493abb-a1f4-533f-2d24-71780a69f247&t=8d02ba2f-f021-560e-034c-effb4dd4b969

Reply:
Yes, thanks!